src/main/java/uk/ac/stfc/facilities/mailing/mailchimp/MailchimpClientConfiguration.java

```diff
@@ -23,9 +23,8 @@ public class MailchimpClientConfiguration {
      */
     public MailchimpClientConfiguration(String apiKey) {
         this.apiKey = apiKey;
-        String dataCenter = apiKey.split(API_KEY_SPLITTER)[1];
-        this.host = dataCenter + "." + MAILCHIMP_API_BASE_HOST;
-        this.baseUri = SCHEME + "://" + host + MAILCHIMP_API_VERSION_URI;
+        this.host = MAILCHIMP_API_BASE_HOST;
+        this.baseUri = SCHEME + "://" + MAILCHIMP_API_VERSION_URI;
```

Comment:
Why have you taken 'host' out from the baseUri? Isn't it just going to read as `https:///3.0/` now?

Reply:
Sorry, the I typed the variable name and selected the wrong variable from the autocomplete menu.